aten/src/ATen/native/README.md

```diff
@@ -191,19 +191,19 @@ more complicated neural network layers (e.g., `conv2d`) and internal functions
 designed specifically for binding (e.g., `cudnn_convolution`).
 
 As we progress along our schema unification of the `func` schema with the JIT
-signatue schema, we must introduce features that allow us to increase compliance.
+signature schema, we must introduce features that allow us to increase compliance.
 One of these features are Tensor annotations. As of now we use naming conventions
 to indicate whether an argument of a function is going to be mutated and returned.
 
 ### `annotations`
 
 There are two typical situations in which we mutate the memory of an argument in the Python
-frontend:
-a) For an inplace operations such as `self.abs_()`
+frontend:  
+a) For an inplace operations such as `self.abs_()`  
```

Comment:
There's some extra white space at the end of this line.

Reply:
Just FYI.

Reply:
It's intentional, two whitespace characters cause the rendering to break to the next line. Otherwise it puts the a) and b) sections all on one line, which looks kind of messy. You can see what it looks like on master here:
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/README.md